modular_zubbers/code/_globalvars/lists/quirks.dm

```diff
@@ -1,19 +1,24 @@
 /// Neccesary for accessing non-instanced quirk whitelists, as initial() cannot access lists
-GLOBAL_LIST_INIT(quirk_species_whitelist, generate_quirk_species_whitelist())
-
+GLOBAL_LIST_INIT(quirk_species_whitelist, generate_quirk_species_lists())
+GLOBAL_LIST_INIT(quirk_species_blacklist, generate_quirk_species_lists())
```

Comment:
These 2 lists have the same content, no idea if this even works, you'd likely need to change it around to accept which list it extracts species ID's from